robottelo/rh_cloud_utils.py

```diff
@@ -83,3 +83,20 @@ def get_remote_report_checksum(org_id):
             continue
         checksum, _ = result.stdout.split(maxsplit=1)
         return checksum
+
+
+def get_report_data(report_path):
+    """Returns report data from tar file.
+
+    Args:
+        tarobj: tar file to get report data from
+    """
+    json_data = {}
+    tarobj = tarfile.open(report_path, mode='r')
```

Comment:
we can use the object as a context manager, in a `with` statement, to close it automatically.

```suggestion
    with tarfile.open(report_path, mode='r') as tarobj:
```

Reply:
Good catch. Updated the pr.